StreamChatClient_v3/Models/Message.swift

```diff
@@ -29,3 +29,13 @@ public struct MessageModel<ExtraData: ExtraDataTypes> {
 public typealias Message = MessageModel<DefaultDataTypes>
 
 public protocol MessageExtraData: Codable & Hashable {}
+
+extension MessageModel: Hashable {
+    public static func == (lhs: Self, rhs: Self) -> Bool {
+        lhs.id == rhs.id
```

Comment:
Do you think it's enough for now?

Reply:
Yes, we talked about it already in some other PR. This is actually the expected behavior.